sourcegraph/sourcegraph
# Vk/add app setup wizard skeleton

Closes https://github.com/sourcegraph/sourcegraph/issues/52123

This PR adds the initial UI and the basic layout changes for the new App specific setup flow. 

## Todo 
- Polish local repositories widget step 
- Check the sign-in flow for the first setup step 

## Test plan
- Check that you can see the setup flow at the very beginning when the app is starting 
- Check that you can sing into the .com and go back to the next setup step 
- Check that you can add local repositories 
- Check that you can finish the flow and see the home page after 
- Check that once you finished the flow you don't see it again next run
